__tests__/tsconfig.json

```diff
@@ -1,8 +1,6 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "module": "es6",
-    "moduleResolution": "node",
```

Comment:
Removed these because they were never actually necessary and I think I copied them over from one of our other TS repos when I converted this library to TS a few months ago.